_posts/2021-07-06-Influential-tech-lead.md

```diff
@@ -0,0 +1,90 @@
+On the other hand, you also need to keep the bigger picture in mind and how the requirement fits into the greater vision.
+If you're a technical team lead that has no vision and isn't going anywhere, then how can you expect your team will follow you?
+In order to create a vision you can ask yourself questions like "What do you want to achieve?, "Which (tangible) output do you want when achieving your goal?", "How would that output change the way things are done now?", ...
+Setting specific goals together with your team that move towards your vision will help you to achieve it.
+
+Once you have that vision and goals, being predictable is key.
+By doing so, everybody knows what you're trying to achieve.
+The Don’t-Repeat-Yourself principle, which is a key principle for each software engineer, does not apply when working with people.
+Communicate your vision and goals over and over again: repetition and predictability are key.
+
+## Be collaborative
+A top-down approach where nobody has a say in the process and where both vision and goals are pushed downwards, is very easy for team members to reject.
+A different and more sustainable approach is to be a collaborative technical team lead.
+This means that you regularly seek out a diversity of opinions and ideas amongst teammates to solve problems.
+Involve your team to create a shared vision, and identify the behaviours necessary to accomplish it.
+It's all about getting the team to think through these difficulties themselves instead of telling them what to do all the time.
+As a result they are more engaged, feel trusted, and are more likely to take ownership of their work.
+Building meaningful relationships is one of the key responsibilities of a technical team lead.
+
+## Practice active listening
+We often think about influence as being what we say or how we say it, but improving your listening skills is key to gaining your team's trust and creating psychological safety.
+As technical team leads we're often pulled in many directions throughout the day. 
+The most common pitfall is that we listen to reply, not to understand.
+It's easy to be a passive listener by multitasking and only listening to the highlights. 
+While it may seem like you don't have time, making the time to really listen to your team can increase your leadership capacity significantly.
+Active listening is the ability to focus completely on your team, understand their message, comprehend the information and respond thoughtfully in a relevant way.
+This also includes not only capturing the message, but paying attention to subtle hints and non-verbal communication such as tone, emphasis, facial expressions and body language.
+
+When your team members know that they will be heard, they are more likely to openly share their ideas and provide honest feedback.
+Knowing how and when to gather knowledge of your team members is important, since you can then synthesize it into a better solution before deciding the course forward.
+
```

Comment:
A true (servant) leader only speaks 20-30% of the time. The rest should be spent on listening

Reply:
good addition! :)